apis/filesystem/src/main/java/org/jclouds/filesystem/config/FilesystemBlobStoreContextModule.java

```diff
@@ -39,7 +39,10 @@
    protected void configure() {
       bind(BlobStore.class).to(LocalBlobStore.class);
       install(new BlobStoreObjectModule());
-      bind(ConsistencyModel.class).toInstance(ConsistencyModel.STRICT);
+      if (System.getProperty("os.name", "").toLowerCase().contains("windows"))
+         bind(ConsistencyModel.class).toInstance(ConsistencyModel.EVENTUAL);
```

Comment:
Agreed, I am mostly exploring options on how to fix the tests on a windows system with a virus checker (very standard setup).